Cargo.toml

```
@@ -17,6 +17,7 @@ anyhow = "1.0.28"
 bincode = "1.2.1"
 clap = "2.33.0"
 dirs = "2.0.2"
+dunce = "1.0"
```

```suggestion
dunce = "1.0.0"
```

Cargo dependencies are a little weird. It's been a while since I had this explained to me, but I think it went something along the line of "1.0 will allow 1.0 and 1.1, while 1.0.0 will allow 1.0.0-1.0.99999...". Since 1.0 -> 1.1 is allowed to be a breaking change under semver, we don't want to allow that.

Even if that's no longer the case (or I'm plain wrong), doesn't hurt to be consistent with the rest of the file.

---

The [Rust docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html) mention how version numbers are handled. I've gone with what seems to be convention.